app.js

```diff
@@ -21,13 +21,13 @@ app.get('/get-folders/:user', (req, res) => {
       } else {
         files.forEach((file) => {
           if (file.isDirectory()) {
-            dirArry.push(file.name);
+            dirArry.push([ file.name, baseDir + '/' + user + '/incoming' ]);
```

Comment:
I think you should add the file.name to the base path folder.
```suggestion
            dirArry.push([ file.name, baseDir + '/' + user + '/incoming' + file.name ]);
```